src/Cms/AppErrors.php

```diff
@@ -180,6 +181,19 @@ protected function getExceptionHookWhoopsHandler(): CallbackHandler
 		});
 	}
 
+	/**
+	 * Initializes a callback handler for logging all fatal errors
+	 *
+	 * @return \Whoops\Handler\CallbackHandler
+	 */
+	protected function getFatalErrorLogWhoopsHandler(): CallbackHandler
+	{
+		return new CallbackHandler(function ($exception, $inspector, $run) {
+			error_log($exception->getMessage());
```

Comment:
I'd use just `error_log($exception)` so that the stack trace is printed as well.